pkg/scheduler/apis/config/testing/policy_test.go

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
We may need to add tests covering JSON->Obj, Obj->{YAML, JSON}. @liggitt Please let me know if they are necessary.